Doc/using/mac.rst

```diff
@@ -25,7 +25,7 @@ there.
 
 What you get after installing is a number of things:
 
-* A :file:`MacPython 3.6` folder in your :file:`Applications` folder. In here
+* A :file:`Python 3.6` folder in your :file:`Applications` folder. In here
```

Comment:
Should be `Python 3.9` for the master branch.

Reply:
I only edit the folder name cause this commit need to be backported to branch 2.7, 3.5, 3.6

Reply:
Should i make another PR for updating this to 3.8 on master branch?

Reply:
Hi @ned-deily , i made a PR for update the version on master branch. See [13966](https://github.com/python/cpython/pull/13966). Thanks for your review!